src/utils/FormattingUtils.ts

```diff
@@ -119,3 +119,13 @@ export function formatCommaSeparatedList(items, itemLimit) {
         return _t("%(items)s and %(lastItem)s", { items: items.join(', '), lastItem: lastItem });
     }
 }
+
+/**
+ * Formats a number into a 'minimal' badge count (9, 99, 99+).
+ * @param count The number to convert
+ * @returns The badge count, stringified.
+ */
+export function formatMinimalBadgeCount(count: number): string {
+    if (count < 100) return count.toString();
+    return "99+";
```

Comment:
Should the value 99 return "99" or "99+"?

Reply:
*asks design*

Reply:
98, then 99+